### ISSUE TITLE
Instructions for training a performance_rnn

### ISSUE THREAD

**Issue by ysig** [tensorflow/magenta#1685]
Hi,

I am trying to run a performance rnn on a jazz dataset, but training doesn't seem at all to generalize. I am using the configuration `performance_with_dynamics`.
Basically the maximum accuracy I get is around 0.37, with a loss of 2.4. Although this can be due to a strange train-val split, the fluctuations of the loss function around the same value for a lot of interactions indicate that my model doesn't seem to learn.

Is this normal? Can you propose some model parametrizations that I can try, in order to make learning more effective?

Thanks in advance!

**Comment by Mannerow** [comment]
I'm having a similar problem. I'm getting a maximum accuracy of about 0.32 on the eval set and 0.85 on the training data. However, I've tried techniques to reduce overfitting such as reducing network size/nodes, lowering dropout_keep_prob, lowering learning rate, and experimenting with num_steps (early stopping). 

Were you ever able to figure out optimal parameters @ysig ??
